include/deal.II/base/symmetric_tensor.h

```diff
@@ -628,6 +628,13 @@ class SymmetricTensor
   SymmetricTensor   operator - () const;
 
   /**
+   * Simple addition of a multiple of a tensor, i.e. *this += a*T,
+   * where the scalar is of the same base type and
+   * T is a symmetric tensor of double.
+   */
+  void add(const Number &a, const SymmetricTensor<rank,dim,double> &p);
```

Comment:
Why only add `SymmetricTensor<rank,dim,double>` and not `SymmetricTensor<rank,dim,Number>`?